src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -125,7 +125,7 @@ import org.joda.time.Duration;
  * </code>
  * </pre>
  */
-@Generated("by GAPIC")
+@Generated("by GAPIC v{settingsFile.fileHeader.generatorVersion}")
```

Comment:
My bad, I forgot to update the baseline after making the change. Now this should be resolved.